scraper/src/compare_datasets.js

```diff
@@ -0,0 +1,120 @@
+import fs from 'fs';
+import { parse as json2csv } from 'json2csv';
+import csv from 'csvtojson';
+import columnMapping from './column_mapping_permapeople.js';
+
+function sanitizeColumnNames(jsonArray) {
+  return jsonArray.map((obj) => {
+    const keys = Object.keys(obj);
+    keys.forEach((key) => {
+      let newKey = key;
+      newKey = newKey.toLowerCase();
+      if (newKey.includes('&amp;')) {
+        newKey = newKey.split('&amp;').join('and');
+      }
```

Comment:
Does this replace all occurrences of `&amp;` with `and`?
I think in this case `newKey.replaceAll('&amp;', 'and')` would make the intention clearer, also the `if` would not be needed.

Reply:
Nice suggestion here! For me personally, a little old `if` is closer to the heart xD However, replaceAll is maybe clearer for others, updated it!